Mlem/Enums/FeedType.swift

```diff
@@ -5,16 +5,34 @@
 //  Created by Jonathan de Jong on 12.06.2023.
 //
 
-import Foundation
+import SwiftUI
 
 enum FeedType: String, Encodable, SettingsOptions {
```

Comment:
*nit*

We have a protocol for associated colors, might as well use it

```suggestion
enum FeedType: String, Encodable, SettingsOptions, AssociatedColor {
```